src/components/PDOKAutoSuggest/PDOKAutoSuggest.tsx

```diff
@@ -40,16 +41,26 @@ export interface PDOKAutoSuggestProps
 const PDOKAutoSuggest: FC<PDOKAutoSuggestProps> = ({
   fieldList = [],
   municipality = configuration.map.municipality,
+  fieldType = configuration.map.fieldType,
   ...rest
 }) => {
-  const fq = municipality
+  const fieldTypeParam = fieldType
+    ? [['fq', `${fieldTypeFilterName}:(${fieldType})`]]
+    : []
```

Comment:
Perhaps an idea to have 'adres' as default, and use fieldType when it is something else than 'adres'

Reply:
The default value of `adres` is set here https://github.com/Amsterdam/signals-frontend/blob/9ba5c6654942fdbf86d1e90041f02be423488ad6/app.base.json#L59